src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/coding/fallthrough/InputFallThrough3.java

```diff
@@ -21,6 +21,7 @@ int hasYield() {
         case 3: // ok
             // fall through
         case 4: // ok
+            yield 5;
```

Comment:
Maybe I missed something, why did we update this input?

Reply:
@nrmancuso may be my laziness updated to kill mutation :)
if you are not fine with that  let me know i will update that

Reply:
I am good, for the future it is a good idea to leave a comment on this yourself and mention why we did this. I assumed killed mutations were related to code changes only m.